front/src/applications/stdcm/utils/formatSimulationReportSheet.ts

```diff
@@ -69,51 +69,61 @@ function timeStringToSeconds(time: string): number {
 /**
  * Based on a stop arrival time and a stop duration, calculate the departure time on this stop
  */
-function computeStopDepartureTime(hhmm: string, mmss: string): string {
+export function computeStopDepartureTime(hhmm: string, mmss: string): string {
```

Comment:
Could you adapt this one also please

```suggestion
/**
 * @param arrivalTime description with time format...
 * @param duration description with time format...
 * @returns description of the return
*/
export function computeStopDepartureTime(arrivalTime: string, duration: string): string {
```

Reply:
done